gem/settings/base.py

```diff
@@ -133,7 +133,7 @@
 
 # We have multiple layouts: use `base`, `malawi` or `springster`
 # to switch between them.
-SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'base')
+SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'springster')
```

Comment:
Please remember to revert this before committing. I think the other option is to run `export SITE_LAYOUT_2='springster'` in your shell before running the server.